ext/snowcrash/src/ResourceGroupParser.h

```diff
@@ -252,12 +252,9 @@ namespace snowcrash
                 return true;
             }
 
-            for (Elements::const_iterator it = blueprint.content.elements().begin();
-                 it != blueprint.content.elements().end();
-                 ++it) {
-
-                if (it->element == Element::CategoryElement
-                    && SectionProcessor<Resource>::isResourceDuplicate(it->content.elements(), uri)) {
+            for (auto it : blueprint.content.elements()) {
```

Comment:
Not an iterator anymore, consider renaming `resourceElementIt`; for example to `element`.